src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -10,6 +10,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
```

Comment:
perhaps we could delete the old prefix_address

Reply:
good catch!